internal/engine/upper_test.go

```diff
@@ -2233,6 +2255,14 @@ func (f *testFixture) setupDCFixture() (redis, server model.Manifest) {
 	return manifests[0], manifests[1]
 }
 
+func (f *testFixture) setDCBuildLogOutput(mn model.ManifestName, output string) {
+	f.b.buildOutput[mn] = output
```

Comment:
again, this doesn't look docker-compose specific

Reply:
That's fair, I'll remove that restriction.

Reply:
(It's now removed)